staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -392,7 +393,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	WatchList: {Default: false, PreRelease: featuregate.Alpha},
 
-	ConsistentListFromCache: {Default: false, PreRelease: featuregate.Alpha},
+	ConsistentListFromCache: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Unit-test failure seems to be lack of:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L658-L665

in cacher

@p0lyn0mial ^^

Reply:
Why we need more testing of etcd storage.

cc @logicalhan @jpbetz @deads2k